docs-src/tut-7-rpc.scrbl

```diff
@@ -183,6 +210,144 @@ invoke our @pyin{main} function.
 
 @(hrule)
 
+Now that we have written an entire @|RPS| game in Python it is time to try
+running it.
+
+First we need to generate TLS certificates.
+Think up a pass phrase greater than four characters long, then open a shell in
+your @tt{rps-python} directory and type @tt{./reach tls}:
+
+@verbatim{
+$ ./reach tls
+*** Warning! Certificates generated by `reach tls` are only suitable ***
+    for development purposes and must not be used in production!
+
+Scaffolding auto-generated TLS certificate + key under ./tls directory...
+(When prompted for a pass phrase, please enter between 4 and 1024 characters)
+
+Generating a RSA private key
+....................................................++++
+.................++++
+writing new private key to './tls/reach-server.key'
+Enter PEM pass phrase:
+Verifying - Enter PEM pass phrase:
+-----
+Signature ok
+subject=CN = reach-server
+Getting Private key
+Enter pass phrase for ./tls/reach-server.key:
+  ...done
+
+When invoking `reach server` you MUST supply the pass phrase you just
+used to generate your certificate as an environment variable, e.g.:
+
+  $ REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
+    REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+    ./reach server
+
+./tls
+├── reach-server.crt
+├── reach-server.csr
+├── reach-server.ext
+└── reach-server.key
+
+0 directories, 4 files
+}
+
+@(hrule)
+
+With those created, now we launch the server:
+
+@verbatim{
+$  REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
+   REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+   ./reach server
+
+===============================
+using rpc key:
+REACH_RPC_KEY=YOUR-PRESHARED-KEY
+===============================
+
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+  Verifying when ONLY "Alice" is honest
+  Verifying when ONLY "Bob" is honest
+Checked 93 theorems; No failures!
```

Comment:
I believe that this will not be shown if they previously compiled, so we shouldn't show it here

Reply:
I see it every time, which I think is a side-effect of how `reach server` currently works.

Reply:
Hmm, interesting. In any case, I think we should drop it from this printout